src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/PublishContent.targets

```diff
@@ -10,13 +10,18 @@
   <!-- gathers the items to be published -->
   <Target Name="GatherItemsForPattern">
     <Error Condition="'$(PublishPattern)' == ''" Text="Please specify a value for PublishPattern using standard msbuild 'include' syntax." />
+
+    <PropertyGroup>
+      <ReleativeBlobPathBase>$(BlobNamePrefix)</ReleativeBlobPathBase>
```

Comment:
It would be nice to also include a port of the other commit in https://github.com/dotnet/buildtools/pull/1494 that fixes this, if there's no big reason not to. It wasn't fixed in `release/1.1.0` either, though, so... I'd say this is a non-blocking comment.

Reply:
Agreed & done